docs/source/release-notes/workarea-3-5-0.html.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Workarea 3.5.0 Release Notes
+excerpt: Release notes for Workarea 3.5.0
+---
+
+Workarea 3.5.0 Release Notes
+======================================================================
+
+The following changes are introduced in Workarea 3.5.0.
+
+_See also:_ [Workarea 3.5.0 Upgrade Guide](/upgrade-guides/workarea-3-5-0.html) and [Workarea Upgrade](https://plugins.workarea.com/plugins/upgrade) (tool you should use for all upgrades).
+
+
+Releases
+----------------------------------------------------------------------
+
+* Improves release previewing by including all releases scheduled to publish before the previewed release
+* Removes release undo functionality in favor of using separate "undo" releases, and adds a workflow for creating them
+* Restores the releases calendar feature in the Admin
+* Allows data file importing in releases
+* Allows featured product changes in releases
+* Allows rules changes in releases
+* Adds a separate Sidekiq queue for releases
+* Changes how releases are represented in Elasticsearch
+
+
+### Pull Requests
+
+* <https://stash.tools.weblinc.com/projects/WL/repos/workarea/pull-requests/3954/overview>
+* <https://stash.tools.weblinc.com/projects/WL/repos/workarea/pull-requests/3982/overview>
+* <https://stash.tools.weblinc.com/projects/WL/repos/workarea/pull-requests/3996/overview>
+* <https://stash.tools.weblinc.com/projects/WL/repos/workarea/pull-requests/4021/overview>
+* <https://stash.tools.weblinc.com/projects/WL/repos/workarea/pull-requests/4092/overview>
+* <https://stash.tools.weblinc.com/projects/WL/repos/workarea/pull-requests/4125/overview>
+* <https://stash.tools.weblinc.com/projects/WL/repos/workarea/pull-requests/4157/overview>
+* <https://github.com/workarea-commerce/workarea/pull/44>
+
+
+Segmentation
+----------------------------------------------------------------------
+
+* Adds segmentation feature to the base platform
+* Adds segment-based metrics and insights
```

Comment:
We'll want another point here to indicate that you can set things things active per-segment to allow to segmented site experiences.